cosmos/migrations/0002_photo_fields.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 3.0.8 on 2020-08-17 06:54
+
+from django.db import migrations, models
+
+import cosmos.models.board
+import cosmos.models.committee
+
+
+class Migration(migrations.Migration):
+
+    replaces = [("cosmos", "0002_photo_fields"), ("cosmos", "0003_auto_20200817_0852")]
+
+    dependencies = [
+        ("cosmos", "0001_init_models"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="board",
+            name="photo",
+            field=models.ImageField(
+                default="boards/default.jpg", upload_to="boards", validators=[cosmos.models.board.validate_aspect_ratio]
+            ),
+        ),
+        migrations.AddField(
+            model_name="committee",
+            name="photo",
+            field=models.ImageField(
+                default="committees/default.jpg",
```

Comment:
![image](https://user-images.githubusercontent.com/19555626/90376247-701e0600-e076-11ea-8bb6-f659acecc470.png)
I'm getting the following error only for creating a committee and not the board. I think this is where it stems from but not quite sure.

Reply:
should be fixed now, file is a png, not a jpg